porch/go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	google.golang.org/grpc v1.44.0
 	google.golang.org/protobuf v1.28.0
 	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
```

Comment:
interesting that we use both versions. Possible to consolidate?

Reply:
Done. Using `gopkg.in/yaml.v2` now.